### ISSUE TITLE
Expo-Payments-Stripe V 7.0.0 Android American Express doesnt work

### ISSUE THREAD

**Issue by rgordon95** [expo/expo#5906]
### Environment

 System:
      OS: macOS 10.14.5
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.1.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^35.0.0 => 35.0.0 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz => 0.59.8 
      react-navigation: ^3.12.0 => 3.13.0 
    npmGlobalPackages:
      expo-cli: 3.1.2

Expo Kit, iOS & Android (this bug is android specific as we are using IAP for iOS but when we used stripe on iOS amex worked)

### Steps to Reproduce

1. Go To Stripe payment interface
2. Click "subscribe" in our app to launch stripe card entry modal
3. Enter an American Express Card (Discover, MC, Visa all tested and work)
4. It detects as valid card, detects proper brand, allows proper entry based on 15 digit length and 4 digit cvv
5. Click Done
6. Unlike other major card providers, nothing happens. No loading indicator. No error message.

### Expected Behavior
1. Go To Stripe payment interface
2. Click "subscribe" in our app to launch stripe card entry modal
3. Enter an American Express Card (Discover, MC, Visa all tested and work)
4. It detects as valid card, detects proper brand, allows proper entry based on 15 digit length and 4 digit cvv
5. Click Done
6. loading indicator triggers, then "Subscription successful" returns.

tl;dr
Payment to process.

### Actual Behavior

"It doesn't work". The request to submit payment processing for american express specifically on android does not appear to be triggered. I know android native modules have specific handling via stripe for each card provider. I suspect there is something with the amex bridge that is not finding the native module for amex on android.

I have tried with a real Amex, as well as both fake stripe test card credentials. It is clear the validation is occuring because if you enter a proper length but incorrect number it will not get to the done function. but the done button is never being enabled.

### Reproducible Demo

https://snack.expo.io/SJ7VZPcOS

**Comment by cruzach** [comment]
And was this working on previous SDKs/version of `expo-payments-stripe`?

**Comment by rgordon95** [comment]
It was not tested in a previous version on android due to needing a higher sdk for android to release an app (needed expo 33+)

**Comment by rgordon95** [comment]
Any update on this?

**Comment by isthisshivam** [comment]
any update on this? Thanks in advance

**Comment by cruzach** [comment]
Hi! Since the `expo-payments-stripe` module was deprecated with the release of SDK 42, I'm going to close this issue.

We collaborated with the team at Stripe to make the `@stripe/stripe-react-native` library work seamlessly in Expo projects, so we recommend you use that library instead. [Learn more in the API reference](https://docs.expo.io/versions/latest/sdk/stripe/), and check out this [excellent YouTube video that walks through integrating Stripe in your Expo app](https://www.youtube.com/watch?v=DZlAET7Tgx4&ab_channel=UnsureProgrammer). Thanks!
